# Represents a gesture recognized from a hand
std_msgs/Header header
bool is_left
string gesture_name            # e.g. "FIST", "THUMB", etc.  